subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts

```diff
@@ -27,21 +29,43 @@ export const democracySummary = async (
 };
 
 export const democracy = async (
```

Comment:
I think this should be modelled in a different way, democracy could return what we get as overview data and referendums/proposals should have their own resolver so it only runs one or the other if a client only needs one. With this approach if you query only proposals it will also execute the code to get the referendums.

Reply:
If everything else is ok with this PR how about we do the remodelling here https://github.com/litentry/api-gateway/issues/54

Reply:
Sure it can be a separate PR, they both can be part of the democracy type, we just need a resolver for proposals and another one for referendums